backend/canvas_app_explorer/models.py

```diff
@@ -1,9 +1,30 @@
+from django.core.exceptions import ValidationError
 from django.db import models
+from django.utils.deconstruct import deconstructible
+from django.utils.html import strip_tags
 from db_file_storage.model_utils import delete_file, delete_file_if_needed
 from tinymce.models import HTMLField
 
-# Create your models here.
+# Based on the built in validators 
+# https://github.com/django/django/blob/main/django/core/validators.py
+# Use in your model as validators=[MaxLenIgnoreHTMLValidator(max_length=120)]
+@deconstructible
+class MaxLenIgnoreHTMLValidator:
+    max_length = 120
```

Comment:
Works like magic! However, can we consolidate the max_length value definition as a constant? Looks like it is also in the database table definition.

Reply:
Yeah, this is just a default value if one isn't passed in via a parameter. The one used in the model is used. 

I was thinking afterward of just raising an exception if it wasn't set or maybe just extending the [MaxLengthValidator](https://github.com/django/django/blob/49b470b9187b6be60e573fed08c8f4a87f133750/django/core/validators.py#L433) that's there.

Reply:
This is outdated with the current code and no longer used.